modules/claims_api/spec/requests/v2/veterans/rswag_power_of_attorney_spec.rb

```diff
@@ -129,281 +129,518 @@
     end
   end
 
-  path '/veterans/{veteranId}/power-of-attorney:appoint-individual' do
-    put 'Appoint an individual Power of Attorney for a Veteran.' do
+  # path '/veterans/{veteranId}/power-of-attorney:appoint-individual' do
```

Comment:
commenting these out for now. We don't intend to use the routes defined here, but we might want to reuse some of the specs.